modules/health_quest/spec/services/appointment_service_spec.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe HealthQuest::AppointmentService do
+  subject { described_class.new(user) }
+
+  let(:user) { build(:user, :health_quest) }
+
+  before do
+    Flipper.enable('show_healthcare_experience_questionnaire')
+    allow_any_instance_of(HealthQuest::UserService).to receive(:session).and_return('stubbed_token')
+  end
+
+  describe '#get_appointment_by_id' do
+    let(:appointment) { subject.get_appointment_by_id(132) }
+
+    it 'is a Hash' do
+      expect(appointment).to be_a(Hash)
+    end
+
+    it 'has a data key of type OpenStruct' do
+      expect(appointment[:data]).to be_a(OpenStruct)
+    end
+
+    it 'data has relevant keys' do
+      expect(appointment[:data].respond_to?(:id)).to be(true)
+      expect(appointment[:data].respond_to?(:start_date)).to be(true)
+      expect(appointment[:data].respond_to?(:clinic_id)).to be(true)
+      expect(appointment[:data].respond_to?(:clinic_friendly_name)).to be(true)
+      expect(appointment[:data].respond_to?(:facility_id)).to be(true)
+      expect(appointment[:data].respond_to?(:sta6aid)).to be(true)
+      expect(appointment[:data].respond_to?(:station_name)).to be(true)
+      expect(appointment[:data].respond_to?(:patient_icn)).to be(true)
+      expect(appointment[:data].respond_to?(:community_care)).to be(true)
+      expect(appointment[:data].respond_to?(:vds_appointments)).to be(true)
+    end
+
+    it 'has pagination information' do
```

Comment:
These values seem odd. Shouldn't they all be positive?

Reply:
In my opinion, I don't think a single appointment should have pagination information. I will confer with the FE team and then either change the values of the pagination section to `1` or just remove that data entirely.

Reply:
After speaking with the FE team I've gone ahead and removed the meta section.